docs/howto/craft-for-platforms/example-c-or-cpp-app.rst

```diff
@@ -0,0 +1,75 @@
+.. _example-c-or-cpp-app:
+
+Example C or C++ app
+====================
+
+This how-to guide covers the steps, decisions, and implementation details that
+are unique when crafting a snap based on C or C++. We'll work through the
+aspects unique to these apps by examining an existing recipe.
+
+The process of developing a snap for a C or C++ app builds on top of standard
+tools like autotools and make, making it possible to adapt or integrate an
+app's existing build tooling into the crafting process.
+
+
+Example recipe for moon-buggy
+-----------------------------
+
+The following code comprises the recipe for the C project `moon-buggy
+<https://github.com/snapcraft-docs/moon-buggy>`_. This project is a small game
+where you drive a buggy across the moon's surface.
+
+.. collapse:: moon-buggy recipe
+
+  .. code:: yaml
+
+    name: moon-buggy
+    base: core22
+    version: '1.0.51'
+    summary: Drive a car across the moon
+    description: |
+        Moon-buggy is a simple character graphics game, where you drive some
+        kind of car across the moon's surface. Unfortunately there are
+        dangerous craters there. Fortunately your car can jump over them!
+    confinement: devmode
+
+    parts:
+        moon-buggy:
+            plugin: autotools
+            autotools-configure-parameters: ["--prefix=/usr"]
+            source: .
+            build-packages:
+                - libncurses-dev
+                - texinfo
+    apps:
+        moon-buggy:
+            command: usr/bin/moon-buggy
```

Comment:
Four-space indentation in YAML looks pretty weird to me. I'd suggest two spaces instead, as I think that's what most people are used to seeing and is what [prettier defaults to](https://prettier.io/docs/en/options#tab-width).

```suggestion
    name: moon-buggy
    base: core22
    version: '1.0.51'
    summary: Drive a car across the moon
    description: |
      Moon-buggy is a simple character graphics game, where you drive some
      kind of car across the moon's surface. Unfortunately there are
      dangerous craters there. Fortunately your car can jump over them!
    confinement: devmode

    parts:
      moon-buggy:
        plugin: autotools
        autotools-configure-parameters: ["--prefix=/usr"]
        source: .
        build-packages:
          - libncurses-dev
          - texinfo
    apps:
      moon-buggy:
        command: usr/bin/moon-buggy
```

Reply:
You're right, I was going to fix this in all the files but got distracted.

Reply:
[342890d](https://github.com/canonical/snapcraft/pull/5160/commits/342890d9e0b2bcb84f35140592fc7c8fe6160b02)